quic/api/QuicTransportBase.cpp

```diff
@@ -1636,10 +1636,19 @@ QuicTransportBase::notifyPendingWriteOnStream(StreamId id, WriteCallback* wcb) {
   if (!qStream->writable()) {
     return folly::makeUnexpected(LocalErrorCode::STREAM_CLOSED);
   }
+
+  if (wcb == nullptr) {
+    return folly::makeUnexpected(LocalErrorCode::INVALID_WRITE_CALLBACK);
+  }
   // Add the callback to the pending write callbacks so that if we are closed
   // while we are scheduled in the loop, the close will error out the callbacks.
-  if (!pendingWriteCallbacks_.emplace(id, wcb).second) {
-    return folly::makeUnexpected(LocalErrorCode::INVALID_WRITE_CALLBACK);
+  auto existing_wcb_pair = pendingWriteCallbacks_.emplace(id, wcb);
```

Comment:
nit: this is a weird name. Can we just call it wcbEmplaceResult?
Also the naming style of the code base for local variables are variableName

Reply:
Done, also added a test